Sources/Data/Model/Event.swift

```diff
@@ -26,11 +26,13 @@ final class Event : NSManagedObject {
     @NSManaged internal(set) var timestamp: Date
     @NSManaged internal(set) var isFlushed: Bool
     
-    override func awakeFromInsert() {
+    public override func awakeFromInsert() {
         // default values for newly created and inserted Events.
+        
         self.id = UUID().uuidString
         self.attributes = Attributes()
         self.deviceSnapshot = DeviceSnapshot()
         self.timestamp = Date()
+        self.isFlushed = false
```

Comment:
This is a no-op because your xcdatamodeld file already sets the default to "NO" (false).